Add blog about OSS Vienna (#311)
xataio/mdx-blog: oss-vienna-happy-hour.mdx

---
title: 'Postgres Happy Hour with Xata at Open Source Summit Europe'
description: 'Join us for the Postgres Happy Hour in Vienna on September 18th!'
image:
  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/postgres-happy-hour.png
  alt: Postgres Happy Hour with Xata at Open Source Summit Europe
author: Gulcin Yildirim Jelinek
authorEmail: gulcin@xata.io
date: 09-03-2024

> tags: ['postgres', 'oss', 'vienna', 'happy-hour', 'xata', 'postgresql', 'postgres-happy-hour', 'ossumit', 'booth']

Reviewer: ```suggestion
tags: ['postgres', 'oss', 'vienna', 'happy-hour', 'xata', 'postgresql', 'postgres-happy-hour', 'ossummit', 'booth']
```

published: true
slug: oss-vienna-happy-hour
ogImage: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/postgres-happy-hour.png
---

## Postgres Happy Hour with Xata at Open Source Summit Europe

On September 16-18th, we'll be at the [Open Source Summit Europe](https://events.linuxfoundation.org/open-source-summit-europe/) in Vienna, Austria.

We will have a booth in the showcase area on the ground floor where we'd love to chat with you about Postgres and Xata.
Personally, this event is extra special for me, as it will be my first time doing booth duty as a new Xata employee.
If you're attending the Open Source Summit, please stop by our booth and say hello!

We are open source enthusiasts here at Xata, and here are a few of our projects:

> - [pgroll](https://github.com/xataio/pgroll): An open source PostgreSQL tool for zero-downtime, reversible schema migrations.

Reviewer: Maybe for this section lets instead link to our OSS feature pages on our website and not Github to help with SEO?

Author: Replaced URLs with website URLs instead of Github repositories.

- [pgstream](https://github.com/xataio/pgstream): An open source change data capture (CDC) command-line tool and library for Postgres.
- [pgzx](https://github.com/xataio/pgzx): An open source framework to easily build PostgreSQL extensions in Zig, a systems programming language for maintainable and fast code.

If you'd like to set up a time to chat about anything related to Postgres, Xata, or open source in general, feel free to email me at gulcin@xata.io.

## Join us for the Postgres Happy Hour!

> It is not only booth duty, but I am happy to announce that we'll be also hosting a [Postgres Happy Hour at the Open Source Summit](https://events.linuxfoundation.org/open-source-summit-europe/features/co-located-events/#postgres-happy-hour-with-xata) on September 18th!

Reviewer: ```suggestion
It is not only booth duty, but I am happy to announce that we'll also be hosting a [Postgres Happy Hour at the Open Source Summit](https://events.linuxfoundation.org/open-source-summit-europe/features/co-located-events/#postgres-happy-hour-with-xata) on September 18th!
```

> At Xata, we are passionate about PostgreSQL and its role in powering modern applications.

Reviewer: I feel like this entire paragraph feels a little too chat GPT like... It's saying a lot without really saying much and is doing so with many common buzzwords. Instead it would be nice to make it more personal as to the reasons why Xata is hosting this happy hour at this event... what are our specific goals / aims?

Author: I don't know how to write this better. We are hosting the event to celebrate Postgres project & community and get to meet people who might be interested in Postgres and Xata. The goal is to explain what we do at Xata to the people who comes. 

How would you write this paragraph? Open to suggestions.

We believe that PostgreSQL is more than just a database; it is a fundamental technology that enables innovation and growth in the tech industry.
And we are excited to celebrate this awesome project with the Postgres community at the Open Source Summit Europe!

> The event is free and open to everyone, even if you are not attending the Open Source Summit.

Reviewer: Is this true? Can someone who doesnt have a summit ticket for the event still get entry into the venue to attend the happy hour? I wouldnt have thought so as usually Linux Foundation events are strict but then maybe this one is an exception?

Author: Yes, it is considered a private event and open to everyone, not only OSS attendees.

However, space is limited to 50 attendees, so please fill out[this form](https://5i8caik7lja.typeform.com/psql-happy-hour) if you'd like to attend.

If you're a Postgres fan and in the area, you're very welcome to join us!

### Why attend?

- **Engaging conversations**: Whether you’re a Postgres expert or curious about how Xata can elevate your next project, connect with like-minded individuals and explore new possibilities.
- **Exclusive swag**: Attendees will receive some cool Xata swag as a token of appreciation.
- **Relaxed atmosphere**: We believe the best ideas flow in a casual setting. Whether you’re diving deep into database architecture or just starting out, expect lively discussions, knowledge sharing, and maybe even a friendly debate or two.

Because talking databases is always better with a drink in hand—come network, learn, and enjoy the Postgres Happy Hour!

In the spirit of celebrating Postgres, we will be adhering to the community's [code of conduct](https://www.postgresql.org/about/policies/coc/).

> ### A personal Postgres story

Reviewer: I think this entire section was not meant to be included :)

Author: We want to adhere to CoC. Which part is not meant be added?

The passage is coherent and reads well, conveying a personal and relatable story. However, a few minor edits can improve the flow and clarity:

1. **Punctuation and Sentence Structure**: Adding commas where needed and breaking down a longer sentence for better readability.
2. **Clarity**: Clarifying the timeline in the first sentence for smoother comprehension.

Here's a revised version:

---

### A Personal Postgres Story

When we were planning the Postgres Happy Hour in Vienna, I realized that it was almost 9 years ago, around the same time of year, that I gave my first PGConf.EU talk in Vienna.
I was overwhelmed by the size of the room and how crowded it felt, despite its spaciousness. It was my first Postgres talk in English, and delivering it at the biggest Postgres conference was terrifying.
The talk was about [Managing Postgres with Ansible](https://slides.com/apatheticmagpie/managing-postgres-with-ansible), which was a cool topic at the time. I remember making so many slides and forgetting to breathe. No other talk seemed so hard after that—it was a big learning experience.

> ![Gulcin and Slonik, in Vienna in 2015](/images/oss-vienna/gulcin-with-slonik-2015.jpg)

Reviewer: This needs to be resized outside (usually just preview on mac) and then re-uploaded so its not so huge :)

Author: Done, let's see how this looks :)

  - _I am always looking for ways to improve documentation and this is right up my alley._
- 11:55 Panel: Why Open Source AI Matters for Europe - Justin Colannino, Microsoft; Sachiko Muto, OpenForum Europe; Stefano Maffulli, Open Source Initiative; Cailean Osborne, The Linux Foundation
  - _As a fellow Europe resident, this is a topic that will personally affect me and I would like to learn more about it._

For lunch this time, I would be joining the **Women and Non-Binary Lunch** at 12:35 and then I plan to do some more booth duty and try to control my excitement until the Postgres Happy Hour!

And finally, the event I look forward to the most: (drumroll) **Postgres Happy Hour with Xata at 17:00!**
I don't need to say more, right? What makes a community is the people in it, I hope we can make some new friends and see some old ones!

## Conclusion

Just by looking at the schedule at Open Source Summit Europe, I felt like a kid in a candy store. Super excited!
If you like to see all the sessions and events, you can check out the [full schedule](https://events.linuxfoundation.org/open-source-summit-europe/program/schedule/).

> I am looking forward to seeing you at the event! If you want to get in touch, please send me an email at gulcin@xata.io.

Reviewer: I would bring back around the call to action to register for the happy hour in this section as well.

Author: Good idea!

---
title: 'Postgres Happy Hour with Xata at Open Source Summit Europe'
description: 'Join us for the Postgres Happy Hour in Vienna on September 18th!'
image:
  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/postgres-happy-hour.png
  alt: Postgres Happy Hour with Xata at Open Source Summit Europe
author: Gulcin Yildirim Jelinek
authorEmail: gulcin@xata.io
date: 09-03-2024
tags: ['postgres', 'oss', 'vienna', 'happy-hour', 'xata', 'postgresql', 'postgres-happy-hour', 'ossummit', 'booth']
published: true
slug: oss-vienna-happy-hour
ogImage: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/postgres-happy-hour.png
---

## Postgres Happy Hour with Xata at Open Source Summit Europe

On September 16-18th, we'll be at the [Open Source Summit Europe](https://events.linuxfoundation.org/open-source-summit-europe/) in Vienna, Austria.

We will have a booth in the showcase area on the ground floor where we'd love to chat with you about Postgres and Xata.
Personally, this event is extra special for me, as it will be my first time doing booth duty as a new Xata employee.
If you're attending the Open Source Summit, please stop by our booth and say hello!

> We are open source enthusiasts here at Xata, and here are a few of our open source projects:

Reviewer 2: ```suggestion
We are open source enthusiasts here at Xata, and here's some of our more popular open source Postgres tools and extensions:
```

Trying to position them as tools vs. projects, but let me know what you think.

Author: Will update, thanks!

- How to Chart Your Own Career Path in Open Source - Ildiko Vancsa, Open Infrastructure Foundation; Nithya Ruff, Amazon; Allison Randal, Capabilities Limited; and Dawn Foster, CHAOSS
- What's the State of Open Source in Europe? Key Research Findings Revealed! - Mirko Boehm, Anna Hermansen, Adrienn Lawson & Cailean Osborne, The Linux Foundation

A fun thing I discovered looking at the schedule: There is a project called Basil and the talk is delivered by what I assumed by two Italians.
So the cutest project name award goes to Basil and if you are curious what is it about search this: "Traceability and Automation Examples With Basil an Open Source Software for Quality Management - Luigi Pellecchia & Gabriele Paoloni, Red Hat"

Then there is a killer slot (12:15) with five talks I could attend, and I know it is not really helpful but here is the list:

<Alert flexDirection="column" alignItems="flex-start">
  <b>12:15 | Demystifying Secure Application Communication with Zero Trust and Istio Without Sidecars</b>

{' '}

<i>Lin Sun, solo.io</i>

  <span style={{ color: 'var(--chakra-colors-textInfo)' }}>

>   </span>

Author: @miukimiu do you think we can make the some brand colors instead of this blue? 

Maybe smth like: 
```
<span style={{ color: '#8566FF' }}>
text
</span>
```

Reviewer 3: I was using blue because we use the brand color as a link. So for accessibility, it is better if we don't mix it. Or the text will look like clickable. For example here, we wouldn't know what is clickable.

<img width="826" alt="Screenshot 2024-09-03 at 15 57 39" src="https://github.com/user-attachments/assets/135ee40c-d892-4225-ba96-db5c7bd68303">

So I changed the color to use the `titleAccent` for a11y instead of the blue. It is a very light color but it works side by side with links.

<img width="828" alt="Screenshot 2024-09-03 at 15 56 00" src="https://github.com/user-attachments/assets/0f0694f1-09ce-4465-b403-b5a2582ee4d6">
